tests/prefetch_related/test_prefetch_related_objects.py

```diff
@@ -33,6 +33,17 @@ def setUpTestData(cls):
         cls.reader1.books_read.add(cls.book1, cls.book4)
         cls.reader2.books_read.add(cls.book2, cls.book4)
 
+        # Initializing models required for generic foreign key tests
+        cls.bookmark1 = Bookmark.objects.create(url="https://www.djangoproject.com/")
+        cls.animal1 = Animal.objects.create(name="lion", weight=100)
+
+        cls.tagged_item1 = TaggedItem.objects.create(
+            tag="great", content_object=cls.bookmark1
+        )
+        cls.tagged_item2 = TaggedItem.objects.create(
+            tag="awesome", content_object=cls.animal1
+        )
+
```

Comment:
These objects are used only in `test_generic_foreign_key`, so we can move creation to the test.